lib/server/validators.ex

```diff
@@ -75,8 +76,8 @@ defmodule Server.Validators do
     |> Result.map(&hd/1)
   end
 
-  @spec conf_password(String.t(), String.t()) :: Result.t(String.t(), String.t())
-  defp conf_password(pass, conf_pass) do
-    Validators.equal_to(pass, conf_pass, "The passwords don't match!")
+  @spec conf_password(Validation.field(any(), a)) :: (a -> Result.t(String.t(), a)) when a: var
+  defp conf_password(pass) do
+    &Validators.equal_to?(&1, pass, "The passwords don't match!")
```

Comment:
Misto `Validators.equal_to?` ma byt `Validators.equal?`.

Reply:
tak jak to mam, tak je to spravne

Reply:
No ve stavajici verzi `Validators` ano. Ale jak koukam na ten PR u https://github.com/iodevs/validatex/pull/7 tak po zacleneni uz to bude [equal?](https://github.com/iodevs/validatex/blob/50631f32d1a326dbb5954a4847950450836a72de/lib/validatex/validators.ex#L218).

Reply:
NN, jsou tam dve fce jedna `equal?`, ktera porovnava hodnotu a pak `equal_to?`, ktera porovnava s fieldem.

Reply:
aha ajaj tak to sry, moje chyba